migrate/helpers.go

```diff
@@ -0,0 +1,174 @@
+package migrate
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/cryptopay-dev/yaga/logger"
+	"github.com/go-pg/pg"
+	"github.com/go-pg/pg/types"
+)
+
+// getTableName for quote table name
+func getTableName() types.ValueAppender {
+	return pg.Q(tableName)
+}
+
+// extractAttributes, such as  version, name and migration-type
+func extractAttributes(filename string) (version int64, name, mType string, err error) {
```

Comment:
mb one regex?

Reply:
I don't think so this is good idea

Reply:
did a lot of unnecessary work instead of executing one command

Reply:
I didn't like regexp

Reply:
very bad